src/components/transactions.rs

```diff
@@ -117,60 +127,52 @@ pub fn sync_from_height(from_height: u32, wallet: &mut Wallet, rpc: &mut dyn Rpc
     };
 
     loop {
-        let block = match rpc.get_block(next_height) {
-            Ok(block) => block,
+        match rpc.get_block(next_height) {
+            Ok(block) => {
+                // if block.prev_hash != wallet.last_block_hash
+                // Fork management is not implemented since block.prev_hash rpc is not yet implemented in Zebra
```

Comment:
just `// Fork management is not implemented`

Reply:
Done